pkg/rules/handlers.go

```diff
@@ -59,21 +64,71 @@ func (a *API) handleList(w http.ResponseWriter, r *http.Request) {
 		}
 
 		w.WriteHeader(http.StatusInternalServerError)
-		json.NewEncoder(w).Encode(rr)
+		_ = json.NewEncoder(w).Encode(rr)
```

Comment:
Is this for the linter?

Reply:
Luckily the linter doesn't complain (my IDE does) but I prefer doing it to explicitly mark that error as ignored, since in case of an error there's no other thing that function can do